kornia/feature/lightglue_onnx/lightglue.py

```diff
@@ -0,0 +1,149 @@
+from __future__ import annotations
+
+from typing import ClassVar
+
+import numpy as np
```

Comment:
this will needs np as dependency -- cc @edgarriba @ducha-aiki

Reply:
Thanks for catching that! I've added a try:except import for numpy too.